multi-agent/lib/mas/elements/nodes/claude_agent/validator.py

```
@@ -31,6 +33,8 @@ class ClaudeAgentNodeValidator(BaseElementValidator):
     via the claude_agent.validate_vertex_connection action (ActionHint).
     """
 
+    VALID_EFFORT_LEVELS = {e.value for e in EffortLevel}
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)